bubble-wrap-style.yaml

```diff
@@ -12,8 +12,96 @@ global:
+            var kind = (global.ux_point_of_view && feature['kind:'+global.ux_point_of_view]) || (global.ux_point_of_view_fallback && feature['kind:'+global.ux_point_of_view_fallback]) || feature.kind;
+            switch( kind ) {
+                case( 'country' ):
+                case( 'indefinite' ):
+                    return true;
+                default:
+                    return false;
+            }
+        }
+    ux_point_of_view_kind_filter_disputed_etc: |
+        function() {
+            // if a ux_point_of_view has been defined use that, else fallback POV, else use the feature's default kind
+            var kind = (global.ux_point_of_view && feature['kind:'+global.ux_point_of_view]) || (global.ux_point_of_view_fallback && feature['kind:'+global.ux_point_of_view_fallback]) || feature.kind;
+            switch( kind ) {
+                case( 'disputed' ):
+                case( 'indeterminate' ):
+                case( 'lease_limit' ):
+                case( 'line_of_control' ):
+                case( 'overlay_limit' ):
+                case( 'disputed_breakaway' ):
+                case( 'disputed_claim' ):
+                case( 'disputed_elusive' ):
+                case( 'disputed_reference_line' ): 
+                    return true;
+                default:
+                    return false;
+            }
+        }
+    ux_point_of_view_kind_filter_disputed_star: |
+        function() {
+            // if a ux_point_of_view has been defined use that, else fallback POV, else use the feature's default kind
+            var kind = (global.ux_point_of_view && feature['kind:'+global.ux_point_of_view]) || (global.ux_point_of_view_fallback && feature['kind:'+global.ux_point_of_view_fallback]) || feature.kind;
+            switch( kind ) {
+                case( 'disputed' ):
+                case( 'line_of_control' ):
+                case( 'disputed_breakaway' ):
+                case( 'disputed_claim' ):
+                case( 'disputed_elusive' ):
+                case( 'disputed_reference_line' ): 
+                    return true;
+                default:
+                    return false;
+            }
+        }
+    ux_point_of_view_kind_filter_unrecognized: |
+        function() {
+            // if a ux_point_of_view has been defined use that, else fallback POV, else use the feature's default kind
+            var kind = (global.ux_point_of_view && feature['kind:'+global.ux_point_of_view]) || (global.ux_point_of_view_fallback && feature['kind:'+global.ux_point_of_view_fallback]) || feature.kind;
+            switch( kind ) {
+                case( 'unrecognized' ):
+                    return true;
+                case( 'unrecognized_country' ):
+                    return true;
+                case( 'unrecognized_region' ):
+                    return true;
+                default:
+                    return false;
+            }
+        }
+    ux_point_of_view_kind_filter_country_capital: |
```

Comment:
@bcamper Can you take a look at this section? When viewing Taipei the capital icon should turn OFF when in the Chinese POV as `country_capital:cn` is **false** (even though the default `country_capital` is **true**). The same pattern we used for the name fallbacks doesn't work here.

Reply:
Believe this is fixed in e5e6f31075230c78fff1d2acf53bd064fa8811b8. The issue is that we don't want the *absence* of a `capital_country:XX` value to turn a capital *off*. We want to check for `!= null` so that we only consider values that are explicitly `true` or `false`.